module/build.sbt

```diff
@@ -5,17 +5,19 @@ name               := "play-googleauth"
 
 organization       := "com.gu"
 
-scalaVersion       := "2.11.8"
+scalaVersion       := "2.12.2"
```

Comment:
Any reason to update this and not the example (which is still stuck on `2.11.7`)? Apart from anything else, means people need to remember the `+` when they do `publishLocal` for dev.